streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -52,8 +51,7 @@
 public class StreamsMetricsImpl implements StreamsMetrics {
 
     public enum Version {
-        LATEST,
-        FROM_0100_TO_24
+        LATEST
```

Comment:
If the enum only has 1 element and named as `LATEST`, did we still need this `enum`?

Reply:
Since we decided to keep the config `built.in.metrics.version` I think it is OK to keep also this enum.